tests/default/data/pdo.php

```diff
@@ -144,15 +144,6 @@ public function updateQuery(PDO $pdo)
         assertType('PDOStatement', $stmt);
     }
 
-    public function leftJoinQuery(PDO $pdo)
```

Comment:
moved into separate file, because our current way of emulating flags does not support aliases